server/reviews.js

```diff
@@ -27,11 +27,12 @@ module.exports = require('express').Router()
   // TO DO: make sure that this review belongs to user and review
```

Comment:
actually I was working on this file too, to add router.params (as we talked with Omri at code review).
I didn't merge my file yet, to avoid any future conflict I would apply your changes to my current branch!
(just so you know :))